docs/resources/address_object.md

```diff
@@ -38,6 +57,17 @@ resource "valtix_address_object" vpc1-ag {
 }
 ```
 
+
++### dynamic address object (group) using a collection of address objects
+```hcl
+resource "valtix_address_object" addr-group-ag {
+  name            = "addr-group-ag"
+  description     = "collection of address groups"
+  type            = "GROUP"
+  address_group_ids = [48, 52]
```

Comment:
can you replace 48 and 52 with address_ids of an address object?

Reply:
ok done